velox/connectors/hive/HiveConnector.cpp

```diff
@@ -182,10 +182,14 @@ std::shared_ptr<common::ScanSpec> HiveDataSource::makeScanSpec(
     const std::vector<const HiveColumnHandle*>& columnHandles,
     memory::MemoryPool* pool) {
   auto spec = std::make_shared<common::ScanSpec>("root");
+  std::set<std::string> partitionKeyNameSet;
```

Comment:
`folly::FastSet`

Reply:
thanks, change to  std::unordered_set<std::string> partitionKeys;